index.html

```diff
@@ -42,8 +42,168 @@ <h1>
       </ul>
 
     </section>
-    
+
   </header>
+
+
+  <main>
+    <!-- Work Section -->
+    <section class="work-section">
+
+      <div class="work-card card-1">
+        <div class="work-card-image-container">
+          <img src="./images/Snapshoot_1.jpg" alt="First work image">
+        </div>
+        <h1>Tonic</h1>
```

Comment:
You have made a very beautiful project portfolio here. 👏 👏 👏 
But I see a few little things you can do to make the project even better. 👇 
- [ ] It's a best practice that a webpage has only one `h1` tag which will show up during an internet search. This practice ensures that your page ranks higher when users search on any search engine. Please replace this `h1` with any other `hx`, could be `h2` or `h3` or any other. Please apply this to the other places that you have used `h1` except in the headline section.